src/dune_pkg/fetch.ml

```diff
@@ -24,8 +24,8 @@ module Curl = struct
       ; "--compressed"
       ; "--user-agent"
       ; Lazy.force user_agent
-      ; "--write-out"
-      ; "%{http_code}\\n"
+      ; "--write-out" (* This arg must be quoted to work on windows *)
+      ; "\"%{http_code}\""
```

Comment:
Weirdly enough probably due to ocamlformat, this looks like it is attached to the wrong line. I really dislike end-of-line comments :(

Reply:
Yes, it is in the "wrong" place, but I'm not sure it matters too much as there is only one quoted argument in proximity anyway.

Reply:
I'll move it anyway.